app/services/proofing/resolution/progressive_proofer.rb

```diff
@@ -155,30 +145,26 @@ def proof_id_address_with_lexis_nexis_if_needed(applicant_pii:, timer:,
 
       def should_proof_state_id_with_aamva?(ipp_enrollment_in_progress:, same_address_as_id:,
                                             should_proof_state_id:, instant_verify_result:,
-                                            residential_instant_verify_result:,
-                                            double_address_verification:)
+                                            residential_instant_verify_result:)
         return false unless should_proof_state_id
-        # If the user is in double-address-verification and they have changed their address then
+        # If the user is in in-person-proofing and they have changed their address then
         # they are not eligible for get-to-yes
```

Comment:
I think John Hooper left this comment. I'm thinking it's not actually relevant anymore since we do `user_can_pass_after_state_id_check?` under the if check? Can't `user_can_pass_after_state_id_check?` help get to yes?